packages/client/src/store/selectors/bookings.ts

```diff
@@ -43,6 +43,22 @@ export const getBookedSlots = (
   state: LocalStore
 ): Record<string, CustomerBookingEntry> =>
   state.firestore.data?.bookedSlots || {};
+/**
+ * Get subscribed slots from state for a specific month
+ * @param state Local Redux Store
+ * @returns record of subscribed slots
+ */
+export const getBookedSlotsByMonth =
+  (month: number) =>
+  (state: LocalStore): Record<string, CustomerBookingEntry> =>
+    Object.entries(state.firestore.data?.bookedSlots || {}).reduce(
+      (acc, curr) => {
+        return DateTime.fromISO(curr[1].date).month === month
+          ? { ...acc, [curr[0]]: curr[1] }
+          : acc;
+      },
+      {}
+    );
```

Comment:
Loving the `Object.entries` usage. I wasn't aware of it at all and it simplifies things a lot. However, to produce more semantic readability, I would destructure the `[key, value]` entires, (rather then using `curr[0]` and `curr[1]`):

```typescript
// Instead of writing
Object.entries(state.firestore.data?.bookedSlots || {}).reduce(
  (acc, curr) => {
    return DateTime.fromISO(curr[1].date).month === month
      ? { ...acc, [curr[0]]: curr[1] }
      : acc;
  },
  {}
);

// I would write
Object.entries(state.firestore.data?.bookedSlots || {}).reduce(
  (acc, [slotId, bookingEntry]) => {
    return DateTime.fromISO(bookingEntry.date).month === month
      ? { ...acc, [slotId]: bookingEntry }
      : acc;
  },
  {}
);
```

Reply:
Also, notice the omitted `{ return /* ... */ }`. It's unnecessary.